app/views/users/new.html.erb

```diff
@@ -55,13 +55,13 @@
         [OpenStruct.new(id: true, name: "Yes"), OpenStruct.new(id: false, name: "No")],
         :id,
         :name,
-        legend: { text: "Are #{pronoun(@user, current_user)} the organisation’s data protection officer?", size: "m" } %>
+        legend: { text: "Is this person the organisation’s data protection officer?", size: "m" } %>
```

Comment:
Any reason not to use `#{perspective(@user, current_user}` here? That way the wording is in a single place?

Reply:
Good point. Will go with the simpler option and use the `perspective` helper on both new and edit views.